DOCS/man/options.rst

```diff
@@ -4395,6 +4395,17 @@ The following video options are currently all specific to ``--vo=gpu`` and
     ``--temporal-dither`` is in use. 1 (the default) will update on every video
     frame, 2 on every other frame, etc.
 
+``--gpu-deint-shader=<mode>``
+    Enable deinterlacing in the gpu fragment shader (default: no). Requires
+    ``--vo=gpu``.
+
+    no
+        Disable deinterlacing.
+    yes
+        Enable deinterlacing of all frames.
+    auto
+        Enable deinterlacing of only frames marked as interlaced.
```

Comment:
The hw deint filters (vdpaupp, vavpp, d3d11vpp) call this `interlaced-only` (although as bool option) for what you call `auto` here. In general it turned out that the bitstream flags are inaccurate, so `--deinterlace` now defaults to inserting those filters in always-deinterlace mode.